etl/steps/data/garden/energy_institute/2023-06-26/statistical_review_of_world_energy.excluded_countries.json

```diff
@@ -0,0 +1,5 @@
+[
+    "Canadian Oil Sands: Total",
```

Comment:
Do the values for Canada and Venezuela not need to be incorporated into regional totals? Just wondering as you keep in "Other *" regions as they are needed to be counted in the regional totals?

Reply:
Good catch. I already checked and, according to a footnote in the corresponding sheet of the excel data file:

> Reserves and R/P ratio for Canada includes Canadian oil sands. Reserves and R/P ratio for Venezuela includes the Orinoco Belt.  Saudi Arabia's oil reserves include NGLs from 2017.